README.md

```diff
@@ -307,5 +307,141 @@ let algodclient = new algosdk.Algod(atoken, aserver, aport);
 algodclient.sendRawTransaction(rawSignedTxn);
 ```
 
+#### Assets
```

Comment:
Consider adding some words. Maybe a a few lines at the beginning. It's a bit out of context.